### PR TITLE
Install dmr-client from source

### PR DESCRIPTION
**Pull request by rido-min**
The dmr-client packages have been deleted.

workflows install the tool by cloning and building manually

Updated to the latest import/export syntax

### CODE REVIEW

File: .github/workflows/push_expanded_to_storage.yml [hunk 1]
```diff
@@ -10,23 +10,30 @@ jobs:
...
-
-      - name: Install resolver client
-        run:  dotnet tool install -g dmr-client --add-source https://nuget.pkg.github.com/Azure/index.json --version 0.0.11
-
       - name: Expand models
         run: |
           for changed_file in ${{ steps.files.outputs.added_modified }}; do
             if [[ ${changed_file} =~ ^dtmi([\/\\][a-z0-9_]+){2,}-([1-9][0-9]{0,8}).json$ ]]; then
-              dmr-client resolve --model-file ${changed_file} --repository $PWD -o ${changed_file//.json/.expanded.json} --silent
+              dmr-client export --model-file ${changed_file} --repository $PWD -o ${changed_file//.json/.expanded.json} --silent
```
Comment (digimaun) [inline .github/workflows/push_expanded_to_storage.yml:36]: `--repo`

File: .github/workflows/validate-models.yml [hunk 2]
```diff
@@ -9,26 +9,33 @@ jobs:
   validate-models:
     runs-on: ubuntu-latest
     steps:
```
Comment (digimaun) [inline .github/workflows/validate-models.yml:11]: I don't think we need logging enabled in the pipeline. The exceptions should explain what went wrong. This is how I have it setup in my fork

https://github.com/digimaun/iot-plugandplay-models/blob/main/.github/workflows/validate-models.yml#L33

Comment (rido-min) [reply .github/workflows/validate-models.yml:11]: I'd expect to have the correct LogLevel by default.

**Review by montgomp — CHANGES_REQUESTED** [review]
I'd prefer one workstream to be making these changes rather than duplicated efforts. The changes in https://github.com/digimaun/iot-plugandplay-models/ seem to be more complete with the scripts we are working to sunset being removed. I think it best to pull the more complete fork when it is completed.

**Review by digimaun — APPROVED** [review]

**Pull request merged**